docs/running-on-yarn.md

```diff
@@ -783,6 +783,14 @@ The following extra configuration options are available when the shuffle service
     NodeManager.
   </td>
 </tr>
+<tr>
+  <td><code>spark.yarn.shuffle.service.logs.namespace</code></td>
+  <td><code>(not set)</code></td>
+  <td>
+    A namespace which will be appended to the class name when forming the logger name to use for 
+    emitting logs from the YARN shuffle service.
+  </td>
```

Comment:
Once we finalize on how we add the namespace to the logger name (per the comment I opened), we should provide an example here of what the final name will look like